classes/Domain/Services/TalkRating/TalkRatingStrategy.php

```diff
@@ -19,4 +19,9 @@ public function isValidRating(int $rating): bool;
      * @throws TalkRatingException
      */
     public function rate(int $talkId, int $rating);
+
+    /**
+     * The name of the rating system, Used in the views to determine what kind of rating system to display
+     */
+    public function getRatingName(): string;
```

Comment:
What do you think about renaming this method to just `name()`?

Reply:
I think getRatingName() is fine or we get rid of the getter and access the property directly